app/src/main/AndroidManifest.xml

```diff
@@ -2,17 +2,26 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
 
+    <uses-feature android:glEsVersion="0x00020000" android:required="true" />
+    <uses-feature android:name="android.hardware.sensor.accelerometer" android:required="true" />
+    <uses-feature android:name="android.hardware.sensor.compass" android:required="true" />
+    <uses-feature android:name="android.hardware.sensor.gyroscope" android:required="true" />
+
     <application
         android:allowBackup="false"
         android:icon="@drawable/ic_launcher"
+        android:isGame="true"
```

Comment:
> Is this necessary? I mean the app isn't really a game and I would prefer to have the app not listed in the play store as game, as it would raise wrong expectations. It also triggers some functionality (game bar), that I don't really know whether we should have it in the app.

Not sure. Actually I took a sample manifest file from LibGDX tutorials, that's why this setting.